.github/workflows/wheels.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+# To publish on every pull request
+# on: [push, pull_request]
+# To publish when a (published) GitHub Release is created, use the following:
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - published
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Used to host cibuildwheel
+      - uses: actions/setup-python@v2
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==2.0.0a4
```

Comment:
Do we really want to pin this to an alpha release?  In fact, do we gain anything from using 2.0, rather than their current stable release (1.11)?

Reply:
hum yes, I used the latest example from their repo, it looks like the latest stable version is 1.12. They recommend using dependabot for keeping this up to date, but  I have never used it: https://cibuildwheel.readthedocs.io/en/stable/faq/#automatic-updates

So I guess, here using 1.12 would  make a lot of sense. I'll change the version